.github/workflows/s3-handler-validations.yml

```diff
@@ -0,0 +1,14 @@
+name: Validate s3-handler
```

Comment:
I may be misunderstanding why this github workflow is here; would these tests instead be handled in our circleci config?

Reply:
Apologies, I was on vacation and just got back.

I'd only added it because I looked at the checks and didn't see any specs running so wanted to make sure my feature branch was good before submitting.

Didn't notice the `.circleci/` directory or how to tap into it to know my feature branch was good, so I added this as I saw other GH Actions were configured.

Happy to put this wherever it needs to be, I just didn't see evidence of them so thought I could add it.